src/main/java/org/apache/commons/net/util/SubnetUtils.java

```diff
@@ -331,6 +331,33 @@ public SubnetUtils(final String address, final String mask) {
         this.broadcast = this.network | ~this.netmask;
     }
 
+    /**
+     * Constructor that takes a CIDR-notation string, e.g. "192.168.0.1/16", a dotted decimal mask and and flag to
+     * return value of {@link SubnetInfo#getAddressCount()}
+     * @param cidrNotation A CIDR-notation string, e.g. "192.168.0.1/16"
+     * @param inclusiveHostCount A boolean value e.g. true
+     * @throws IllegalArgumentException if the parameter is invalid,
+     * i.e. does not match n.n.n.n/m where n=1-3 decimal digits, m = 1-2 decimal digits in range 0-32
+     */
+    public SubnetUtils(final String cidrNotation, final boolean inclusiveHostCount) {
```

Comment:
using constructor overloading and setting inclusiveHost count.

Reply:
Feels superflous since there is a setter method. We do not want, IMO, to add more contructor variants for this and that setting. So -1 from me. Let's see what others think.

Reply:
Agreed. 

It would have been better to add these variants instead of creating the setter, but that is done now, and it's not possible to make the class final now without breaking the API.